crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/staticdata/enums/CurrentStatus.java

```diff
@@ -17,16 +18,21 @@ public enum CurrentStatus {
     COMPLETE("COMPLETE", "Complete");
 
     @NotNull
-    @JsonPropertyDescription("This will have the current status")
-    private String status;
-    private String description;
+    @JsonPropertyDescription("This will have the frequency code of the selection")
```

Comment:
Is the description change intentional, seems to be a copy paste error? This should be 'current status' and not frequency code.

Reply:
Yep, well spotted! :)